app/views/users/show.html.erb

```diff
@@ -1,3 +1,44 @@
 <div class="container">
-  <h1> Here is a list of a specific created user </h1>
+  <h2> Users profile number <%= params[:id] %> </h2>
+  <section>
+
+    <article>
+      <img src="<%= @user.photo %>">
+    </article>
+
+    <article>
+      <h1> <%= @user.name %> </h1>
+
+      <p> Number of posts: <%= @user.posts_counter %> </p>
+    </article>
+
+  </section>
+
+  <section>
+    <h2> Bio </h2>
+
+    <p> <%= @user.bio %> </p>
+  </section>
+
+  <section>
+    <% @user.three_most_recent_posts.each do |post| %>
```

Comment:
- [ ] Also getting an error on the user's show page. Please fix this.

![Screenshot from 2022-05-04 03-02-49](https://user-images.githubusercontent.com/26767383/166570141-d1811bfd-0ea8-4745-b47f-205b8c404eb0.png)